packages/gatsby-cli/src/reporter/redux/types.ts

```diff
@@ -6,7 +6,7 @@ export interface IGatsbyCLIState {
   activities: {
     [id: string]: IActivity
   }
-  status: string
+  status: ActivityStatuses | string
```

Comment:
Do we need ` | string` part? It feels like `status` should only have `ActivityStatuses` enum/union (same comment applies to other place where typing like that was added)

Reply:
Yes! As I didn't want to change behavior this was needed to fix TS type errors.

We're using an empty string on multiple places for the default state: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/src/reporter/loggers/ink/components/develop.tsx#L52

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/src/reporter/redux/reducer.ts#L8

Reply:
Wanted to see first if we want to change this behavior to set it to e.g. `ActivityStatus.NotStarted` by default

Reply:
Worth exploring using different default/fallback. In case there are problems - we could set type to `ActivityStatuses | ""` (more strict than `string`)

Reply:
For this PR - maybe just make it stricter (`ActivityStatuses | ""`). And we can figure out using different fallback/default in another PR (it's not really concern of this PR IMO). What do you think about that?

Reply:
Yeah, sounds like a good idea!